browser.ts

```diff
@@ -0,0 +1,38 @@
+import { fromParse5 } from 'hast-util-from-parse5';
```

Comment:
Couldn’t you access the DOM in browsers?
In which case, https://github.com/syntax-tree/hast-util-from-dom is very light and fast

Reply:
Yes we can! Thanks for the suggestion.